js/metric.js

```diff
@@ -41,6 +42,17 @@ export class Metric {
     this.htmlName = htmlText
   }
 
+  updateDescription (newDescription) {
+    if (newDescription === undefined) {
+      window.MetricQWebView.instances[0].handler.metricQHistory.metadata(this.htmlName).then((metadataObj) => {
+        this.description = metadataObj.description
+        this.htmlName += ' '
```

Comment:
This seems like a very brittle workaround. Is it actually necessary or can this just be removed?